Gopkg.toml

```
@@ -21,11 +21,11 @@
 #  version = "2.4.0"
 
 
-[[constraint]]
-  branch = "master"
+[[override]]
   name = "github.com/giantswarm/microerror"
+  version = "=0.1.0"
```

```suggestion
  version = "v0.1.0"
```

Why does this not work?

---

I think we need `=` because the minor version `v0.2.0` introduced an API breaking change, which was not smart.